interview-questions.md

```diff
@@ -5,46 +5,124 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
+1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! 
+I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the 
```

Comment:
The actual primary key from the cohort model will become the foreign key for the student model. 
cohort_id: integer